blocks/lists.js

```diff
@@ -643,6 +643,50 @@ Blockly.Blocks['lists_getSublist'] = {
   }
 };
 
+
+Blockly.Blocks['lists_sort'] = {
+  /**
+   * Block for sorting a list.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      "id": "list_sort",
+      "message0": Blockly.Msg.LISTS_SORT_TITLE,
+      "args0": [
+        {
+          "type": "field_dropdown",
+          "name": "TYPE",
+          "options": [
+            [Blockly.Msg.LISTS_SORT_TYPE_NUMERIC, "numeric"],
```

Comment:
Our convention is to use all uppercase for the language-neutral dropdown constants.